test/fail_compilation/test21198.d

```diff
@@ -0,0 +1,24 @@
+// https://issues.dlang.org/show_bug.cgi?id=21198
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/test21198.d(23): Error: copy constructor `test21198.U.this` cannot be used because it is annotated with `@disable`
```

Comment:
This error message is quite confusing because the ctor is generated. Could you add an `errorSuplemenental` stating that it's caused by `s` having a copy ctor?

Reply:
I agree the error is confusing, but I think it makes more sense to file it as a separate enhancement request, since it affects all generated ctors, not just the ones related to issue 21198.

Reply:
Submitted as [issue 21204](https://issues.dlang.org/show_bug.cgi?id=21204).

Reply:
Fair enough